src/components/Pagination.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <section class="container mx-auto p-10 md:py-10 px-5 md:p-10">
+    <ul class="flex align-items-center justify-center pagination-items">
+      <li @click="loadFirstPage" class="first filter drop-shadow-md border border-gray-200">First</li>
```

Comment:
I would wrapp all of your pagination elements in a `<button>` tag to allow the natural behavior of a clickable element, i.e. cursor adjustments, opportunity to disable the natural click event as needed, etc.

Reply:
I completely agree. Buttons will be better.